src/diffusers/loaders.py

```diff
@@ -1815,6 +1818,7 @@ def from_single_file(cls, pretrained_model_link_or_path, **kwargs):
         # import here to avoid circular dependency
         from .pipelines.stable_diffusion.convert_from_ckpt import download_from_original_stable_diffusion_ckpt
 
+        config_file = kwargs.pop("config_file", None)
```

Comment:
Can this be renamed to `original_config_file` so that it matches the argument name in `download_from_original_stable_diffusion_ckpt`

Reply:
done